internal/signalfx-agent/bundle/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN ENCODED_VER=$(echo $JDK_VERSION | sed 's/_/%2B/g') && \
     else \
     OPENJDK_URL="${OPENJDK_BASE_URL}/jdk-${ENCODED_VER}/OpenJDK11U-jdk_${ARCH}_linux_hotspot_${JDK_VERSION}.tar.gz"; \
     fi && \
-    wget -nv -O /tmp/openjdk.tar.gz "$OPENJDK_URL"
+    wget -nv --no-check-certificate -O /tmp/openjdk.tar.gz "$OPENJDK_URL"
```

Comment:
Please revert this change and other occurrences of that change, as we want to check certs for our downloads as much as possible.